### ISSUE TITLE
typo on readme.md

### ISSUE THREAD

**Issue by Mocha--** [fi3ework/vite-plugin-checker#403]
### Describe the bug

The readme file name is a typo. It is [README.md](https://github.com/fi3ework/vite-plugin-checker/blob/main/README.md)

### Reproduction

N/A

### Expected behavior

README.md

### System Info

```shell
N/A
```

### Validations

- [X] Read the [docs](https://github.com/fi3ework/vite-plugin-checker#readme).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by jhoermann** [comment]
I don't see any problem with the filename but currently the readme is missing on npm: 
![image](https://github.com/user-attachments/assets/9a4116d5-9f68-4e45-91d8-8348408cad54)

Seems like `npm run release` wasn't triggered before the release which copies the readme file to the vite-plugin-checker package @fi3ework.

**Comment by Mocha--** [comment]
my bad. I was wondering why there is no readme preview on npm.

**Comment by fi3ework** [comment]
@jhoermann Thanks for reminding!
